vscode-extension/src/extension.ts

```diff
@@ -2,11 +2,31 @@
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from "vscode";
 
+import { exec, spawn } from "child_process";
+import path from "path";
+import { EXTENSION_NAME, COMMANDS } from "./util";
+
 // This method is called when your extension is activated
 // Your extension is activated the very first time the command is executed
 export function activate(context: vscode.ExtensionContext) {
   // Use the console to output diagnostic information (console.log) and errors (console.error)
   // This line of code will only be executed once when your extension is activated
+  console.log(
+    `Congratulations, your extension ${EXTENSION_NAME} is now active!`
+  );
+
+  // Create an output channel for the extension
+  const extensionOutputChannel = vscode.window.createOutputChannel("AIConfig", {
+    log: true,
+  });
+
+  let setupCommand = vscode.commands.registerCommand(COMMANDS.INIT, () => {
+    installDependencies(context, extensionOutputChannel);
```

Comment:
If any of the required steps fail during initialization (e.g. no pip), doesn't this still activate the extension (albeit with messages from the initialization)? Would it be better to fail the activation if we know it will probably have issues due to missing dependencies?